utils/cliutils/commandsflags.go

```diff
@@ -1327,6 +1337,18 @@ var flagsMap = map[string]cli.Flag{
 		Name:   "format",
 		Hidden: true,
 	},
+	Filestore: cli.BoolFlag{
+		Name:  Filestore,
+		Usage: "[Default: false] Set to true to make the transfer mechanism check for the existence of artifacts in the filestore. Used when the database was loaded separately and was not indexed.` `",
+	},
+	IncludeRepos: cli.StringFlag{
+		Name:  IncludeRepos,
+		Usage: "[Optional] Specifies repositories to include in the transfer. You can use wildcards to specify multiple repositories and specify multiple repositories/patterns by separating between them with a semicolon.` `",
+	},
+	ExcludeRepos: cli.StringFlag{
+		Name:  ExcludeRepos,
+		Usage: "[Optional] Specifies repositories to exclude from the transfer. You can use wildcards to specify multiple repositories and specify multiple repositories/patterns by separating between them with a semicolon.` `",
```

Comment:
Specifies repositories to exclude from the transfer. You can use wildcards to specify multiple repositories and specify multiple repositories/patterns by separating between them with a semicolon.
-->
A list of semicolon separated repositories to exclude from the transfer. You can use wildcards to specify patterns for the repositories' names.